OrbitService/CaptureServiceImpl.cpp

```diff
@@ -12,11 +13,51 @@ namespace orbit_service {
 using orbit_grpc_protos::CaptureRequest;
 using orbit_grpc_protos::CaptureResponse;
 
+namespace {
+
+using orbit_grpc_protos::CaptureEvent;
+
+class CaptureResponseListenerImpl final : public CaptureResponseListener {
```

Comment:
Could this have a more descriptive name? Like `GrpcCaptureResponseListener` or `GrpcCaptureResponseSender` or something like that?

Reply:
yes - this is better, renamed.